en/components/grid_toolbar.md

```diff
@@ -77,7 +81,7 @@ export class AppModule {}
 When the user initiates an export process the IgxGrid's `onToolbarExporting` event is emitted. This event exposes both the export options and the exporter itself. This allows you to configure the export process.
```

Comment:
We should add some intro, for example, 'In addition to simply switching on and off the toolbar features, the export process can be further configured in the `onToolbarExporting` event which is emitted when the user initiates the export process.'

Reply:
Done